battleship.py

```diff
@@ -163,17 +183,32 @@ def isHorizontal(ship):
 Returns: list of ints
 '''
 def getClickedCell(data, event):
-    return
+    x = int(event.y/data["cell"])
+    y = int(event.x/data["cell"])
+    clickedCell = []
+    clickedCell.append(x)
+    clickedCell.append(y)
+    return clickedCell
 
 
 '''
 drawShip(data, canvas, ship)
 Parameters: dict mapping strs to values ; Tkinter canvas; 2D list of ints
 Returns: None
-'''
+
 def drawShip(data, canvas, ship):
     return
+'''    
 
+def drawShip(data, canvas, ship):
+    cellsize = data["cell"]
+    for row in range(data["rows"]):
+        for col in range(data["cols"]):
+            grid = [row,col]
+            if grid in ship:
+                canvas.create_rectangle(col*cellsize,row*cellsize,(col+1)*cellsize,(row+1)*cellsize,fill="white",width=1)
+            else :
+                canvas.create_rectangle(col*cellsize,row*cellsize,(col+1)*cellsize,(row+1)*cellsize,fill="blue",width=1)  

```

Comment:
Try writing this function by only drawing the ship not the rest of the grid (only white part not blue) and check if that works.

Reply:
@Pragya28 Removed Else part from code still, it is working fine. Please review again.